jOOQ-examples/jOOQ-spring-boot-example/pom.xml

```diff
@@ -11,7 +11,7 @@
 
     <groupId>org.jooq</groupId>
     <artifactId>jooq-spring-boot-example</artifactId>
-    <version>${org.jooq.version}</version>
+    <version>${jooq.version}</version>
```

Comment:
Yeah, that could happen :) But that's also the reason they recommend overwriting the managed properties, if you need a special version. If jOOQ 3.17.349 has a new module which is included via the Bill of Materials, you won't have the correct version if you set org.jooq.version to whatever.
Yeah, dependencies are fun.